jupyter/nmm-game-utils.ipynb

```diff
@@ -331,58 +347,68 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Returns the coordinates of all mills the given player has\n",
-    "# Set of Frozensets of Tuples(ring, cell)\n",
     "def findMills(board, p):\n",
     "    # Calculate all mills on the rings\n",
     "    return {\n",
     "        frozenset((r, (c+o)%8) for o in range(0, 3))\n",
-    "        # Iterate over all rings\n",
     "        for r in range(0, 3)\n",
-    "        # Iterate over all corners\n",
     "        for c in range(0, 8, 2)\n",
-    "        # All 3 following cells starting at the given corner have to belong to the player\n",
     "        if all(\n",
     "            cell == p\n",
-    "            # Iterate over all 3 cells of the given side (c) by appending the first element\n",
-    "            # as the last mill wrapps around\n",
     "            for cell in (list(board[r]) + [board[r][0]])[c:c+3]\n",
     "        )\n",
     "    # Calculate all mills crossing the rings\n",
     "    } | {\n",
     "        frozenset((r, c) for r in range(0, 3))\n",
-    "        # Iterate over cells in the middle of a side\n",
     "        for c in range(1, 8, 2)\n",
-    "        # All 3 cells in the middle of a given side have to belong to the player\n",
     "        if all(\n",
     "            board[r][c] == p\n",
-    "            # Iterate over all 3 rings\n",
     "            for r in range(0, 3)\n",
     "        )\n",
     "    }"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Die Funktion `countMills` berechnet für einen Spielbrett, eine Menge bereits existierender Mühlen und einen Spieler, die Anzahl der Mühlen, die neu entstanden sind. Die Funktion hat 3 Parameter:\n",
+    "\n",
+    "* `board` $\\in \\{s[1] | s \\in States\\}$\n",
+    "* `oldMills` ist eine Menge von Mühlen (Menge von Koordinaten)\n",
```

Comment:
Also entweder hier auch ekelhafte Schreibweise oder drüber eine humanere verwenden (I like this much more!)

Reply:
Naja, also gemischt haben wir die Schreibweisen sonst ja auch. Deswegen halte ich das nicht unbedingt für ein Problem.

Reply:
klar, ist auch eher meine generelle Abneigung für die Schreibweise beim `board` :D

Reply:
Da habe ich oben noch was vorgeschlagen, ich bastel mal gerade einen Screenshot :)

Reply:
@nclskfm Hat den Streit geklärt :rocket: